minimal/compose/certbot/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM palobo/certbot:latest
```

Comment:
@pslobo we should use tags here to ensure nothing breaks when we update the image